src/components/InfoBox/InfoBox.tsx

```diff
@@ -3,7 +3,15 @@ import _ from 'lodash';
 import { Popper } from 'react-popper';
 import { replaceAllRgbWithHex } from '../../utils/colorConversion';
 
-import styles from '../RedLiner/RedLiner.module.scss';
+let styles = require('../RedLiner/RedLiner.module.scss');
+
+if (!styles) {
+  try {
+    styles = require('../RedLiner/RedLiner.css');
+  } catch (err) {
+    console.error(`Failed to load RedLiner.css: `, JSON.stringify(err));
+  }
+}
```

Comment:
Trying to figure out if this will work (doesn't look like it). If not, then looking for another solution.